seeds/courses.js

```diff
@@ -790,4 +790,9 @@ exports.seed = function (knex) {
         }
       ])
     })
+    .then(() => {
+      return knex.raw(
```

Comment:
Could separate into own commit as this seems like its own bugfix (fix id sequence because inserts set explicit ids which does not increment sequence) 😁

Reply:
splitted